### PR TITLE
Create versioning.md

### PR DESCRIPTION
**Pull request by rowa97**

### CODE REVIEW

**Comment by marbemac** [comment]
Sweet, @nick to add to and polish once versioning is pretty much ready to go. 

In particular: 

- [ ] describe the two primary usage patterns
  - when the project is primarily meant for general documentation, one will likely just want to use one version (not releases) for each major version of the product the docs describe. This will lead a SMALL amount of major versions (1.0, 2.0, etc). For example, Tivo has version 1 and version 2 of their product and docs to match. Note that you can also maintain these two very different versions in different stoplight projects, but then they cannot be presented in the same published docs (would be different domains). This is preferred if the user is willing to publish to separate domains.
  - when the project is connected to a product/api release cycle, use versions + releases. You'll create stoplight project versions/releases to mirror your product/api releases, as they happen. You'll have as many versions/releases in stoplight as you do in your product. This will lead to a LARGE amount of releases (1.0.0, 1.0.1, 1.1.0, 1.2.0, 2.0.0, 2.0.1, 2.0.2, etc).
- [ ] what is the difference between a version and a release? when to use which?
- [ ] note that releases are immutable (cannot be changed)
- [ ] recommended best practices

File: articles/versioning/introduction.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+## Best Practices
+
+Depending on the focus of your project, we recommend following different versioning patterns for the best work flow experience within Stoplight. For example, if you project's primary focus is documentation you may want to use a different approach than if the focus was modeling out an API, but feel free to use whatever pattern you feel works best for your team!
+
+### Product/API Release Cycle
+
+The simpliest approach is when your project is connected to and follows your product's release cycle. We suggest that you version your project by mirroring that cycle accordingly. This means that you should release a project, everytime you also release your project/API. Likewise, you should create a new version in your project when you are ready to start working on new or "breaking" changes to your product. This workflow is more sequential and will lead to a large amount of versions and releases.
+
+### Different Workspaces
```
Comment (casserni) [inline articles/versioning/introduction.md]: @rowa97 Maybe think of a better title based on the content of this section

File: articles/versioning/introduction.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Versions are editable snapshots of your current Project. When you create a new Project, it will automatically start at version 1.0. [More on Versions...](./versions.md)
+
+### Releases
+
+If you want to take a permanent time stamped snapshot of a version, you can create a Release. Once a Release is created, it can be viewed but is no longer editable. [More on Releases...](./releases.md)
+
+## Best Practices
+
+Depending on the focus of your project, we recommend following different versioning patterns for the best work flow experience within Stoplight. For example, if you project's primary focus is documentation you may want to use a different approach than if the focus was modeling out an API, but feel free to use whatever pattern you feel works best for your team!
```
Comment (marbemac) [inline articles/versioning/introduction.md]: - "Work flow -> workflow."
- "if you -> if your"
- "focus was modeling -> focus is modeling" (use consistent tense)

File: articles/versioning/introduction.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+### Product/API Release Cycle
+
+The simpliest and most common approach is to mirror your product or APIs release cycle. Everytime you release your product, you release your Project. Likewise, you should create a new version of your project when you are ready to start working on new or "breaking" changes to your API. This workflow is more sequential and will lead to a large amount of versions and releases.
```
Comment (marbemac) [inline articles/versioning/introduction.md]: - "Everytime you release your product, you release your Project." -> "The basic rule of thumb is that when you release a new version of your API, you create a new release in Stoplight as well under the same version number."
- remove likewise sentence
- "This workflow is more sequential and will lead to a large amount of versions and releases." -> "This release flow is one-to-one with your API, and will usually result in a large number of releases within Stoplight."

File: articles/versioning/introduction.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+### Different Workspaces
+
+Sometimes a project may not follow a typical product release cycle. For example, you may have a project where you want to manage multiple workspaces. For this case, we suggest creating one version for each major workspace, in the start, and then many releases to track the growth and progress of each. When using this approach, it is important to note that you will not be able to move work back and forth between each version, unless you copy it over manually. The benefit being that it allows your team to manage different versions of a project at the same time and then document them under the same domain when publishing.
```
Comment (marbemac) [inline articles/versioning/introduction.md]: ### Platform Versions

Re-write to indicate if its a documentation driven project, trying to describe the few major versions of your platform, can just use a few major versions to describe it all. Don't need to use releases in this approach. For example, "Stoplight Classic" and "Stoplight Next" have two completely different sets of docs, so might represent them as two versions.

File: articles/versioning/releases.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+# Releases
+
+## What
+
+If you want to take a permanent time stamped snapshot of a version, you can create a Release. Once a Release is created, it can be viewed but is no longer editable. You should create releases when you want to mark your work as completely done. These are good for tracking the history and growth of your project.
```
Comment (marbemac) [inline articles/versioning/releases.md]: These are useful to tie your Stoplight project to a specific sprint or product release cycle.

File: articles/versioning/versions.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
+# Versions
+
+## What
+
+Versions are editable snapshots of your current Project. When you create a new project, it will automatically start at version 1.0. You should create a new version within your project when you are ready to start working on large or "breaking" changes to your product. This allows for managing multiple workspaces,while still keeping the older ones availble to edit.
```
Comment (marbemac) [inline articles/versioning/versions.md]: - space after the comma
- replace workspaces verbiage
- make sure user understands that will usually have very few versions per project

**Comment by rowa97** [comment]
changes made

**Pull request merged**